devices/patches/usb_otg/fs_l4+_global.yaml

```diff
@@ -0,0 +1,272 @@
+GOTGCTL:
+  _add:
+    VBVALOEN:
+      description: VBUS valid override enable
+      bitOffset: 2
+      bitWidth: 1
+    VBVALOVA:
+      description: VBUS valid override value
+      bitOffset: 3
+      bitWidth: 1
+    AVALOEN:
+      description: A-peripheral session valid override enable
+      bitOffset: 4
+      bitWidth: 1
+    AVALOVAL:
+      description: A-peripheral session valid override value
+      bitOffset: 5
+      bitWidth: 1
+    BVALOEN:
+      description: B-peripheral session valid override enable
+      bitOffset: 6
+      bitWidth: 1
+    BVALOVAL:
+      description: B-peripheral session valid override value
+      bitOffset: 7
+      bitWidth: 1
+    EHEN:
```

Comment:
This is called HNGSCS in rm0432. Is this intended?

Reply:
It is bit 12. HNGSCS is bit 8

Reply:
Oh sorry, my bad